server/api/batch_inventory.py

```diff
@@ -689,22 +701,29 @@ def download_batch_inventory_ballot_manifest(
     csv_io = io.StringIO()
     ballot_manifest = csv.writer(csv_io)
 
-    ballot_manifest.writerow(["Container", "Batch Name", "Number of Ballots"])
-
-    # We originally didn't have counting group stored, so we make this
-    # optional for backwards compatibility
+    # We originally didn't have a batch_to_counting_group key at all, so we protect against the key
+    # not existing by using .get
```

Comment:
Though the key always exists now, the value can still be `None`. That's why I reworded a bit here

Reply:
If the key always exists, why the `.get`? Backward compatibility?

Reply:
Exactly. If we were to open an old audit created before we added this field and try to download a ballot manifest through this tool, the backend would 500.

Though typing this out, I don't foresee a case where this would happen in prod. Since, once an audit is completed, jurisdiction admins no longer see the batch inventory tool. Just a screen indicating that the audit is complete.

Could happen in dev if we were to re-open an old audit for testing, though

Reply:
Added back the phrase "backward compatibility" for clarity